src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -38,47 +46,101 @@
+                {getTestFile("testUpdatePGT.gvcf"), getTestFile( "testUpdatePGT.output.vcf"), NO_EXTRA_ARGS, b37_reference_20_21},   //testUpdatePGT
+                {getTestFile("gvcfExample1.vcf"), getTestFile( "gvcfExample1.vcf.expected.vcf"), NO_EXTRA_ARGS, b37_reference_20_21}, //single sample vcf
+                {getTestFile("gvcfExample1.vcf"), getTestFile( "gvcfExample1.vcf.expected.vcf"), Arrays.asList("-L", "20"), b37_reference_20_21}, //single sample vcf with -L
+                {getTestFile("combined_genotype_gvcf_exception.original.vcf"), getTestFile( "combined_genotype_gvcf_exception.output.vcf"), NO_EXTRA_ARGS, b37_reference_20_21}, //test that an input vcf with 0/0 already in GT field is overwritten
+                {getTestFile("combined_genotype_gvcf_exception.nocall.vcf"), getTestFile( "combined_genotype_gvcf_exception.output.vcf"), NO_EXTRA_ARGS, b37_reference_20_21},  //same test as above but with ./.
+                {getTestFile(basePairGVCF), getTestFile( "ndaTest.expected.vcf"), Collections.singletonList("-nda"), b37_reference_20_21},  //annotating with the number of alleles discovered option
+                {getTestFile(basePairGVCF), getTestFile( "maxAltAllelesTest.expected.vcf"), Arrays.asList("-maxAltAlleles", "1"), b37_reference_20_21 }, //restricting the max number of alt alleles
+                {getTestFile(basePairGVCF), getTestFile( "standardConfTest.expected.vcf"), Arrays.asList("-stand_call_conf","300"),b37_reference_20_21}, //set minimum calling threshold
+                {getTestFile("spanningDel.combined.g.vcf"), getTestFile( "spanningDel.combined.g.vcf.expected.vcf"), NO_EXTRA_ARGS, b37_reference_20_21},
+                {getTestFile("spanningDel.delOnly.g.vcf"), getTestFile( "spanningDel.delOnly.g.vcf.expected.vcf"), NO_EXTRA_ARGS, b37_reference_20_21},
+                {getTestFile("spanningDel.depr.delOnly.g.vcf"), getTestFile( "spanningDel.depr.delOnly.g.vcf.expected.vcf" ), NO_EXTRA_ARGS, b37_reference_20_21},
+                {getTestFile("ad-bug-input.vcf"), getTestFile( "ad-bug-output.vcf"), NO_EXTRA_ARGS, b37_reference_20_21}, //Bad AD Propagation Haploid Bug
+                {getTestFile("CEUTrio.20.21.gatk3.4.g.vcf"), getTestFile( "CEUTrio.20.21.expected.vcf"), Arrays.asList("--dbsnp", "src/test/resources/large/dbsnp_138.b37.20.21.vcf"), b37_reference_20_21},
+                {getTestFile("CEUTrio.20.21.missingIndel.g.vcf"), getTestFile( "CEUTrio.20.21.missingIndel.expected.vcf"), Arrays.asList("--dbsnp", "src/test/resources/large/dbsnp_138.b37.20.21.vcf"), b37_reference_20_21},
+                {new File(largeFileTestDir + "gvcfs/cutDown.24_sample.21.g.vcf"), new File( largeFileTestDir + "gvcfs/cutDown.24_sample.21.expected.vcf"), NO_EXTRA_ARGS, b38_reference_20_21},
+                {getTestFile("chr21.bad.pl.g.vcf"), getTestFile( "chr21.bad.pl.expected.vcf"), Arrays.asList("-L", "chr21:28341770-28341790"), b38_reference_20_21},
```

Comment:
Some expected outputs here are versioned (as, eg., coming from a particular version of GATK 3.x) while others are not. Where did the unversioned ones come from, and can we record their provenance in their names?